src/uikit.cs

```diff
@@ -700,6 +700,18 @@ public partial interface NSTextStorage {
 		[Since (7,0)]
 		[Notification, Internal, Field ("NSTextStorageDidProcessEditingNotification")]
 		NSString DidProcessEditingNotification { get; }
+
+		// we need to hide this methods inherited from the NSMuttableAttributedString since the class does not implement them
```

Comment:
Minor typo in comment: NSMu**tt**ableAttributedString -> NSMu**t**ableAttributedString